cmd/juju/application/refresh.go

```diff
@@ -503,6 +512,78 @@ func (c *refreshCommand) Run(ctx *cmd.Context) error {
 	return nil
 }
 
+// checkSetSeries determines whether an update to the application's Series
+// is required, and returns the new series, or "" if update not required.
+//
+// If upgrading from a pod-spec (v1) charm to sidecar (v2), override the
+// application's series to what it would be for a fresh sidecar deploy.
+func (c *refreshCommand) checkSetSeries(
+	apiRoot api.Connection,
+	currentSeries string,
+	chID application.CharmID,
+	switchCfg bool,
+) (newSeries string, err error) {
+	logger.Criticalf("TODO: applicationInfo.Series=%q", currentSeries)
```

Comment:
Obviously need to remove these logs.

Reply:
Yeah, forgot to mention that I'm leaving these in temporarily just to help with manual testing. Will take them out soon.

Reply:
Done.